### ISSUE TITLE
Installing HeidiSQL updates resets shortcuts

### ISSUE THREAD

**Issue by mbomb007** [HeidiSQL/HeidiSQL#941]
## Steps to reproduce this issue

1. Set a custom shortcut in Tools -> Preferences -> Shortcuts.
2. Update HeidiSQL.
3. The shortcut is gone.

## Current behavior

I updated HeidiSQL and the shortcut I'd bound for Undo = Ctrl+Y was gone.

## Expected behavior

Updates preserve Preferences

## Environment

* **HeidiSQL version:** 10.3.0.5918 (64-bit)
  
* **Operating system:** Windows 10

**Comment by ansgarbecker** [comment]
That should not be the case, even not with updates between most major versions. Please report the old version you were using while updating.

**Comment by mbomb007** [comment]
Is there a way to find out what it was before updating?

**Comment by ansgarbecker** [comment]
Not really. But as the build update is disabled if there's a major release, and you say you did a build update to 10.3.x, then it was definitely something after the official v10.3.0.5771 . Or did you download an installer?

**Comment by mbomb007** [comment]
I followed the instructions for reporting an issue here, which says to check for updates. I went to "About" in the menu and checked for updates, then installed them.

**Comment by ansgarbecker** [comment]
My question is, was it an installer, asking you for the path where to install and such things. Or was it an entirely automated process (which then was a build update)?
Or, in other words: did you click the upper or the lower button?
![grafik](https://user-images.githubusercontent.com/7986591/76795641-fed9de80-67c9-11ea-9c33-11c695bc4842.png)

**Comment by mbomb007** [comment]
Lower button. It was a build with an automated process.

**Comment by ansgarbecker** [comment]
I was just testing again but could not reproduce the issue.

Please shout if it still persists  for you.
